plugins/chat/app/services/service/base.rb

```diff
@@ -327,9 +327,10 @@ def run!
     end
 
     # @!visibility private
-    def fail!(message)
+    def fail!(messages)
```

Comment:
Made this change so that we can pass multiple errors here in case of validation errors from creating a model, see the CreateCategoryChannel service